tests/vm_scheduling/test_vm_scheduling_scenario.py

```diff
@@ -123,5 +127,46 @@ def test_config(self):
         expected = 1130
         self.assertEqual(expected, pm_amount)
 
+
+class TestPriceModel(unittest.TestCase):
+
+    def setUp(self):
+        config_path = "tests/vm_scheduling/env_config.yml"
+        with io.open(config_path, "r") as in_file:
+            raw_config = yaml.safe_load(in_file)
+            config = convert_dottable(raw_config)
+        env = Env(
+            scenario=config.env.scenario,
+            topology=config.env.topology,
+            start_tick=config.env.start_tick,
+            durations=config.env.durations,
+            snapshot_resolution=config.env.resolution
+        )
+        metrics, decision_event, is_done = env.step(None)
+        idx = 0
+        while not is_done:
+            if idx == 1000:
+                break
+            idx += 1
+            action = AllocateAction(
+                vm_id=decision_event.vm_id,
+                pm_id=decision_event.valid_pms[0]
+            )
+            self.metrics, decision_event, is_done = env.step(action)
```

Comment:
image that you are designing the test cases. If I told your final metrics are wrong, the thing you need to do is using a toy case (so that you can know the output of every step), and compare the output of the program to the correct output. 

Not run the whole program to check if it is aligned with your current output. You never know whether your current output is correct or not.